pub/dependabot-pub.gemspec

```diff
@@ -1,7 +1,5 @@
 # frozen_string_literal: true
 
-require "find"
```

Comment:
I'm curious: Was this simply an unused import or was it necessary somehow?

If unused, I'm surprised we don't have a rubocop linter in place to catch this...

Reply:
Yeah, unused.

I don't think RuboCop provides anything like that. I guess it could implement something based on convention, but it would have a lot of false positives. A `require` can bring in any top level constants or any other side effects, so static analysis does not seem like a good fit to detect this kind of thing.

Reply:
Gotcha. 

It's a common linter in the python and go ecosystems, both of which can have side effects as well, but there are conventions for signaling if you expect an import to have side effects.